src/ServiceControl.UnitTests/Archiving/ArchiveGroupTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using NUnit.Framework;
+using Raven.Client;
+using ServiceControl.Contracts.Operations;
+using ServiceControl.MessageFailures;
+using ServiceControl.Recoverability;
+using ServiceControl.UnitTests.Recoverability;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ServiceControl.UnitTests.Archiving
+{
+    [TestFixture]
+    public class ArchiveGroupTests
+    {
+        [Test]
+        public void ArchiveGroup_skips_over_empty_batches_but_still_completes()
+        {
+            // Arrange
+            using (var documentStore = InMemoryStoreBuilder.GetInMemoryStore())
+            {
+                var groupId = "TestGroup";
+
+                using (var session = documentStore.OpenSession())
+                {
+                    var secondAchiveBatch = new ArchiveBatch { Id = ArchiveBatch.MakeId(groupId, ArchiveType.FailureGroup, 1) };
```

Comment:
Would it make sense to pull this Id out as a `emptyBatchId` or something? It is used below as well.

Reply:
👍